e2e/test/iothub/service/ServiceClientE2ETests.cs

```diff
@@ -34,33 +35,37 @@ public async Task Message_NoTimeoutPassed()
 
         private async Task FastTimeout()
```

Comment:
I think these tests are BS anyway. They were originally about the AMQP-required overloads that took a timeout instead of a cancellation token. Now we don't timeout that way.